### PR TITLE
Account generation page

### LINKED ISSUE #307: Account Generation Page
We have account generation instructions on the [Claims page](claims.polkadot.network) and some walkthrough on the Kusama guide. We need to migrate this information into its own dedicated page on the wiki so we have somewhere to link people to when they ask how to generate an account.

### PR DESCRIPTION
**Pull request by lsaether**
closes #307 

basically just copy / pastes the content from the claims page to the wiki

### CODE REVIEW

**Comment by Swader** [comment]
Can you try rebasing if this was made before the link checker merge so we can see if it's okay for internal links checking?

File: docs/learn-account-generation.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+Click “Save” and your account will be created. It will also generate a backup file that you should save to
+your computer. Ideally, you would also save it on an external hard drive or thumb drive, or print it out and
+be able to re-enter it later. You should not store it in cloud storage, email it to yourself, etc.
+
+You can use this backup file to restore your account. This backup file is not readable unless it is decrypted
+with the password.
+
+![pjs-05](assets/accounts/polkadot_js_05.jpg)
+
+## Coinbase
```
Comment (laboon) [inline docs/learn-account-generation.md]: Let's remove the Coinbase custody information here.  It's not really specifically about "account generation".

File: docs/learn-account-generation.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+The polkadot.js plugin will then use system randomness to make a new seed for you and display it to you in the
+form of twelve words.
+
+You should back up these keys. It is imperative to store the seed somewhere safe, secret, and secure. If you
+cannot access your account via polkadot.js for some reason, you can simply re-enter your seed, and have access
+to your account once again.
+
+![plugin-05](assets/accounts/polkadot_plugin_js_05.jpg)
+
+### Store the Key Safely
```
Comment (laboon) [inline docs/learn-account-generation.md]: We have the same information about securing the key safely in multiple places.  I think we should just move this to one place at the top.  It made more sense on the claims page when people were taking different "paths" but here it just seems redundant.

**Review by laboon — CHANGES_REQUESTED** [review]
Two minor changes, otherwise looks good

**Review by laboon — APPROVED** [review]

**Pull request merged**